packages/create-block/lib/init-wp-scripts.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+const { command } = require( 'execa' );
+const { install } = require( 'pkg-install' );
```

Comment:
What's the value in using `pkg-install` here, vs. `command`, which we already have in scope anyways, and with which we're doing the `npm run build` command. Couldn't we `command( 'npm install' )` as well? I note that `pkg-install` is [documented as using `execa` under the hood](https://www.npmjs.com/package/pkg-install).

Is the main benefit that it supports Yarn?

Reply:
Yes, in the current shape we probable can avoid using `pkg-install` and go with:
```js
await command( 'npm install @wordpress/scripts --save-dev', {
 	cwd,
 } );
```

I'll give it a try.